recipes/yaff/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "yaff" %}
+{% set version = "1.4.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 7bd303f8a2c34d139cbba830b18ebfadb890d8a45f48e160c58dc66c974bba78
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+  skip: True  # [not linux or py<37]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+    - numpy
+    - cython >=0.26
+    - matplotlib >1.0.0
+    - h5py >=2.0.0
+    - molmod >=1.4.1
+    - scipy >=0.17.1
```

Comment:
Above 4 dependencies are not needed in `host`

Reply:
But they are listed in the `install_requires`: https://github.com/molmod/yaff/blob/master/setup.py

Reply:
You only need the `setup_requires` dependencies in `host` and `install_requires` dependencies are needed in `run`.

Reply:
Thx